auth.php

```diff
@@ -127,11 +132,8 @@ public function canDo($cap) {
                         // assume we want profile info.
                         return $this->chained_auth->canDo($cap);
                     }
-// I don't know how to handle "external" in this context yet.
-// Is it in any way sensible to mix regular auth with external auth? 
-//                case 'external':
-//                    //We are external if one of the chains is valid for external use 
-//                    return $this->trustExternal($_REQUEST['u'],$_REQUEST['p'],$_REQUEST['r']);
+                case 'external':
+                    return $this->chained_auth->canDo($cap);
```

Comment:
This case should be added a bit further up, because it returns the same value as they do...

Reply:
fixed